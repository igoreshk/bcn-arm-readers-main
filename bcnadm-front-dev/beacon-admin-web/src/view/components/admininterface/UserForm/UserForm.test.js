import React from 'react';
import { shallow } from 'enzyme';

import { UserForm } from './index';

const props = {
  currentUser: {},
  handleSubmit: jest.fn(),
  history: {},
  rolesMap: new Map(),
  translate: jest.fn(),
  valid: true
};

const wrapper = shallow(<UserForm {...props} />);

describe('UserForm', () => {
  it('Check matching snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
  it('Check component has rendered', () => {
    expect(wrapper.length).toEqual(1);
  });
});
