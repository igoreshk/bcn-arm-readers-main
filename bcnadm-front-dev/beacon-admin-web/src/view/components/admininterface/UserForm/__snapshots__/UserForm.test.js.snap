// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserForm Check matching snapshot 1`] = `
<WithStyles(ForwardRef(Dialog))
  className="userFormDialog"
  open={true}
>
  <WithStyles(ForwardRef(DialogTitle)) />
  <WithStyles(ForwardRef(DialogContent))>
    <form
      className="userForm"
    >
      <div
        className="userFormLeft"
      >
        <Field
          component={[Function]}
          name="firstName"
        />
        <Field
          component={[Function]}
          name="lastName"
        />
        <Field
          component={[Function]}
          name="email"
        />
        <Field
          component={[Function]}
          name="status"
        />
      </div>
      <div
        className="userFormRight"
      >
        <label />
        <Field
          component={[Function]}
          name="role"
        />
        <strong />
      </div>
    </form>
  </WithStyles(ForwardRef(DialogContent))>
  <WithStyles(ForwardRef(DialogActions))>
    <WithStyles(ForwardRef(Button)) />
    <WithStyles(ForwardRef(Button))
      color="primary"
      disabled={false}
    />
  </WithStyles(ForwardRef(DialogActions))>
</WithStyles(ForwardRef(Dialog))>
`;
